### PR TITLE
Add data fetching, refactor initial centering logic, tweaks

### PR DESCRIPTION
**Pull request by chadeddington**
### Changes
* Added iron-ajax
* Added properties for building the request
* Added checks for a supplied path
* ~~Added logic to center the chart on target relative~~
* Redesign of center on load
* Remove hardcoded familysearch

**Previous Commits**
* build reqUrl
* create graph from response
* fix styles import
* Add the iron-ajax dependency, build the path with data based on the 3 different end points
* Make matrix property private, add comments, listeners for when the relationId gets set
* Reset positioning when rebuilding. Center column with relation
* Reset positioning when rebuilding. Center column with relation
* Add fetch and tryRequest if no user id
* Fix y centering and clean up

@jshcrowthe

### CODE REVIEW

File: birch-relationship-calc.html [hunk 1]
```diff
@@ -7,13 +7,15 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
-
+<!-- <script src='../fetch/fetch.js'></script> -->
```
Comment (jshcrowthe) [inline birch-relationship-calc.html]: Should this be commented out?

File: demo/index.html [hunk 2]
```diff
@@ -32,13 +32,64 @@
...
         });
 
-        fetch('./response.json')
-        .then(function(res) {
-          return res.json() 
-        })
-        .then(function(data) {
-          relationshipCalc.relationshipPath = data.data.persons
-        })
+        relationshipCalc.relationshipPath = [ {
```
Comment (jshcrowthe) [inline demo/index.html]: Why did we change the demo?

File: bower.json [hunk 3]
```diff
@@ -22,7 +22,9 @@
     "wc-i18n": "jshcrowthe/wc-i18n",
     "fs-button": "jshcrowthe/fs-button",
     "pinch-gesture": "chadeddington/pinch-gesture",
-    "cedar-dialog": "familySearchElements/cedar-dialog"
+    "cedar-dialog": "familySearchElements/cedar-dialog",
+    "oak-ajax-behavior": "familysearchelements/oak-ajax-behavior",
+    "iron-ajax": "~1.2.0"
   },
```
Comment (jshcrowthe) [inline bower.json]: `fetch` isn't listed in the bower.json

File: birch-relationship-calc.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
-<!-- <script src='../fetch/fetch.js'></script> -->
+<script src='../fetch/fetch.js'></script>
```
Comment (jshcrowthe) [inline birch-relationship-calc.html]: I remember this was having issues, does this work?

File: demo/index.html [hunk 5]
```diff
@@ -32,64 +32,13 @@
...
-        }, {
-          "personId" : "KWZC-4YW",
-          "name" : "Gerdie Cropper Reynolds",
-          "gender" : "FEMALE",
-          "relationshipToPrevious" : "WIFE",
-          "living" : null,
-          "top" : null
-        } ]
-
+        fetch('./response.json')
```
Comment (jshcrowthe) [inline demo/index.html]: :+1: Yayyyy for `fetch`!

File: birch-relationship-calc.html (hunk 4 above)
Comment (chadeddington) [inline birch-relationship-calc.html]: @jshcrowthe I tested this in safari and it appeared to be working there.

**Pull request merged**